src/Sarif.Viewer.VisualStudio/SarifToolWindow.cs

```diff
@@ -22,7 +22,7 @@ namespace Microsoft.Sarif.Viewer
     /// </para>
     /// </remarks>
     [Guid("ab561bcc-e01d-4781-8c2e-95a9170bfdd5")]
-    public class SarifToolWindow : ToolWindowPane
+    public class SarifToolWindow : ToolWindowPane, IToolWindow
```

Comment:
This is being called by other classes that are being used in UnitTests.  Implement IToolWindow to enable this class to be mocked.